pkg/execution/state/redis_state/key_generator.go

```diff
@@ -136,6 +139,10 @@ func (d DefaultKeyFunc) PauseIndex(ctx context.Context, kind string, wsID uuid.U
 	return fmt.Sprintf("%s:pause-idx:%s:%s:%s", d.Prefix, kind, wsID, event)
 }
 
+func (d DefaultKeyFunc) Invoke(ctx context.Context) string {
+	return fmt.Sprintf("%s:invoke", d.Prefix)
```

Comment:
shouldn't need anymore suffix since the correlationID should be guaranteed to be unique, and collusion shouldn't happen

Reply:
Should we create maps by function ID?  In the event of account deletions/cleanups/etc this will provide better namespacing for easy cleanup.

Reply:
Done at [2776a77](https://github.com/inngest/inngest/pull/1265/commits/2776a77df72979e9cd00443fa0954f42dfe9fc86)